src/erasure-code/clay/ErasureCodeClay.cc

```diff
@@ -334,62 +341,63 @@ int ErasureCodeClay::minimum_to_repair(const set<int> &want_to_read,
   int lost_node_index = (i < k) ? i : i+nu;
   int rep_node_index = 0;
 
-  //add all the nodes in lost node's y column.
+  // add all the nodes in lost node's y column.
   vector<pair<int, int>> sub_chunk_ind;
   get_repair_subchunks(lost_node_index, sub_chunk_ind);
   if ((available_chunks.size() >= (unsigned)d)) {
-    for (int j = 0; j < q; j++) {  
+    for (int j = 0; j < q; j++) {
       if (j != lost_node_index%q) {
         rep_node_index = (lost_node_index/q)*q+j;
         if (rep_node_index < k) {
           minimum->insert(make_pair(rep_node_index, sub_chunk_ind));
-        }
-        else if (rep_node_index >= k+nu) {
+        } else if (rep_node_index >= k+nu) {
           minimum->insert(make_pair(rep_node_index-nu, sub_chunk_ind));
         }
       }
     }
-    for (set<int>::iterator i = available_chunks.begin();
-	   i != available_chunks.end(); ++i) {
-      if (minimum->size() < (unsigned)d) {
-        if (minimum->find(*i) == minimum->end()) minimum->insert(make_pair(*i, sub_chunk_ind));
-      } else break;
+    for (auto chunk : available_chunks) {
+      if (minimum->size() >= (unsigned)d) {
+	break;
```

Comment:
space needed?

Reply:
@mynaramana it's a tab. because we use a tab-width of 8 and github's web interface does not respect the editor variable or our CodingStyle, the indent does not look right. but if you checkout the code, and open it using VIM or Emacs. it should look sane.